### ISSUE TITLE
Included in the newest SDK

### ISSUE THREAD

**Issue by tujoworker** [yagitoshiro/ImageAsResized#10]
Looks like Android has this on the newest SDK versions as well: http://docs.appcelerator.com/titanium/latest/#!/api/Titanium.Blob-method-imageAsResized

**Comment by yagitoshiro** [comment]
Yes, but those bundled function sometimes crushes on lower versions of android devices so some people are still using this module on their apps, as far as I know.
